.cirrus.yml

```diff
@@ -34,7 +34,7 @@ task:
 task:
   name: Build (FreeBSD)
   freebsd_instance:
-    image_family: freebsd-13-2
+    image_family: freebsd-14-2
```

Comment:
FreeBSD 13 is still supported, the currently version is 13.5. I think this should be `freebsd-13-5`.

Reply:
That's better yes, it seems to have a longer EoL https://endoflife.date/freebsd.